integrationtest/processor_test.go

```diff
@@ -94,6 +94,84 @@ func TestErrorCallback(t *testing.T) {
 	}
 }
 
+func TestHeaders(t *testing.T) {
```

Comment:
test was just moved from examples-package to the integration tests, because it does not demonstrate how to use the tester-framework but actually tests something.